.github/workflows/js-client-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: 'js-client'
```

Comment:
I added this one following our subgraph and contracts workflows. 

Moreover, can we find a better name than `js-client` and change it everywhere?
Should it be named `sdk`?